### ISSUE TITLE
Localize more user facing strings

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/jupyter-chat#49]
### Problem

Briefly looking at the code, it looks like there are a couple more strings could be localized using the JupyterLab translator, for example: 

https://github.com/jupyterlab/jupyter-chat/blob/f3ba6e06ea09740303a9c9ceeeea494e5fcc63f7/packages/jupyterlab-collaborative-chat/src/index.ts#L341

### Proposed Solution

Use the `ITranslator` in more places.

**Comment by brichet** [comment]
@jtpio would it be enough to use the `ITranslator` ?
Don't we have to provide also the translations for each sentence somewhere (i.e. in language packs directly) ?

**Comment by jtpio** [comment]
Yes it's enough to just use the `ITranslator` for the extension code itself :+1: 

But I think we also need to add it to this repo: https://github.com/jupyterlab/language-packs?tab=readme-ov-file#adding-a-new-extension, so it can be integrated in the translation process via crowdin.

When using the `translator` in the extension, we can then use the name of the extension, for example:

```
translator.load('jupyter-chat')
```
